libs/net/entities/AVOverviewInstance.cs

```diff
@@ -0,0 +1,55 @@
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.Json;
+using TNO.Core.Data;
+
+namespace TNO.Entities;
+
+/// <summary>
+/// AVOverviewInstance class, provides a DB model to manage different types of overviews.
+/// </summary>
+[Cache("av_overview_instance")]
```

Comment:
We don't need to cache the instances.  An instance represents a published report.